app/bundles/CampaignBundle/Tests/Command/ExecuteEventCommandTest.php

```diff
@@ -8,7 +8,7 @@ class ExecuteEventCommandTest extends AbstractCampaignCommand
 {
     public function testEventsAreExecutedForInactiveEventWithSingleContact()
     {
-        putenv('CAMPAIGN_EXECUTIONER_SCHEDULER_ACKNOWLEDGE_SECONDS=1');
```

Comment:
I think that these changes should be reverted. Read the last comment in https://www.php.net/manual/en/reserved.variables.environment.php#114811:

> Please note that writing to $_ENV does not actually set an environment variable, i.e. the variable will not propagate to any child processes you launch (except forked script processes, in which case it's just a variable in the script's memory). To set real environment variables, you must use putenv().

However, in https://www.php.net/manual/en/function.putenv.php the first comment states:

> putenv/getenv, $_ENV, and phpinfo(INFO_ENVIRONMENT) are three completely distinct environment stores. doing putenv("x=y") does not affect $_ENV; but also doing $_ENV["x"]="y" likewise does not affect getenv("x").

So I'm wondering whether we should just go with `new Dotenv(true);` to keep the current behavior.

Reply:
I think I found the issue, `getenv` and `putenv` are not thread safe, this why the got eliminated by `dotenv`. The issue is that the test env loader is not loading custom params. I trying to fix this

Reply:
I reverted to use `getenv` and putenv`, my solution did not work as expected.